File: Java/Ex006.java
/*
Ex006 Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada.
 */
package ex006;

import java.util.Scanner;

public class Ex006 {
   

    public static void main(String[] args) {
    
        Scanner t = new Scanner(System.in);
        System.out.println("Digite um número: ");
        float num = t.nextFloat();
        float dobro = num * 2;
        float triplo = num * 3;
        float raiz = (float) Math.sqrt(num);
        System.out.printf("Você digitou o número: %.0f %n", num);
        System.out.printf("Que quando dobrado fica no valor de: %.0f %n", dobro);
        System.out.printf("E quando triplicado tem o valor de: %.0f %n", triplo);
        System.out.printf("Tendo sua raiz quadrada em %.1f", raiz);
        
    
    }
    
}

File: Python/ex006.py
#Ex006 Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada.
import math
n = int (input('Digite um número: '))
d = n * 2
t = n * 3
r = math.sqrt(n)

print(f'O dobro de {n} é {d} \nO seu triplo é: {t} \nE sua raiz quadrada é {r:.1f}')
